config.yaml

```diff
@@ -118,11 +140,6 @@ repositories:
     tsc: read
   visibility: public
 - name: liboqs
-  collaborators:
-    Martyrshot: write
```

Comment:
@Martyrshot seems to be deleted as a collaborator but not added to a team.

Reply:
[He is already a member](https://github.com/open-quantum-safe/tsc/blob/6dc9cd8c6c443e005f0526e5215f054c3554c7f1/config.yaml#L21) of l[ib-oqs-committers](https://github.com/open-quantum-safe/tsc/blob/6dc9cd8c6c443e005f0526e5215f054c3554c7f1/config.yaml#L136) which maintains his access.

Reply:
Whoops, I missed that. Sorry.

Reply:
Thanks for looking out for me @dstebila 😄